src/arvgvdevice.h

```diff
@@ -29,11 +29,33 @@
 
 G_BEGIN_DECLS
 
+#ifdef FAST_HEARTBEAT
```

Comment:
Defines should be prefixed ARV_, or ARAVIS if they come from the configure script.

In this case, ARAVIS_FAST_HEARTBEAT should be defined by ./configure. See how it is done for ARAVIS_BUILD_USB or ARAVIS_BUILD_VIEWER in configure.ac .